src/main/java/top/seraphjack/simplelogin/client/command/CommandRegister.java

```diff
@@ -0,0 +1,43 @@
+package top.seraphjack.simplelogin.client.command;
+
+import mcp.MethodsReturnNonnullByDefault;
+import net.minecraft.command.CommandBase;
+import net.minecraft.command.CommandException;
+import net.minecraft.command.ICommandSender;
+import net.minecraft.server.MinecraftServer;
+import top.seraphjack.simplelogin.client.Notifier;
+import top.seraphjack.simplelogin.network.MessageLogin;
+import top.seraphjack.simplelogin.network.NetworkLoader;
+
+import javax.annotation.ParametersAreNonnullByDefault;
+
+@ParametersAreNonnullByDefault
+@MethodsReturnNonnullByDefault
+public class CommandRegister extends CommandBase {
+    @Override
+    public String getName() {
+        return "register";
+    }
+
+    @Override
+    public String getUsage(ICommandSender sender) {
+        return "/register <Password> <Password>";
+    }
+
+    @Override
+    public void execute(MinecraftServer server, ICommandSender sender, String[] args) throws CommandException {
+        if (args.length != 2) {
```

Comment:
Same as previous one

Reply:
no, or how to split args when registering?

Reply:
The same as how wpmcmc handles space(